packages/manager/apps/web/client/app/hosting/multisite/git-integration/removal/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,14 @@
+{
+  "hosting_multisite_git_association_removal_ask_confirmation": "Souhaitez-vous vider le contenu du répertoire <b>{{ path }}</b> ?",
+  "hosting_multisite_git_association_removal_warning_title": "Suppression de Git",
+  "hosting_multisite_git_association_removal_warning_description": "Nous allons supprimer l'association avec Git.",
+  "hosting_multisite_git_association_delete_success_message": "Votre suppression a bien été prise en compte.",
+  "hosting_multisite_git_association_delete_danger_message": "Une erreur est survenue: {{ error }}.",
+  "hosting_multisite_git_association_removal_warning_description_delete_dir": "Nous allons supprimer l'association avec Git et le contenu du répertoire.",
+  "hosting_multisite_git_association_removal_confirmation_title": "Supprimer l'association de Git",
+  "hosting_multisite_git_association_confirm": "Confirmer",
+  "hosting_multisite_git_association_removal_next": "Suivant",
+  "hosting_multisite_git_association_removal_abort": "Annuler",
```

Comment:
```suggestion
```

I think no need we can reuse common wizard translation already exsiting for next an cancel

Reply:
Need the next since I am using a ternary to override data-primary-label
![image](https://github.com/ovh/manager/assets/8767496/71c2f2bf-2978-4b7a-a7ca-b187b917e67c)